Make tracing parameter configurable on ingress GW (#8714)
hashicorp/consul: website/content/docs/connect/config-entries/ingress-gateway.mdx

        ' or will default to the `default` namespace.',
      yaml: false,
    },
    {
      name: "TracingStrategy",
      type: `string: "random_sampling"`,

>       yaml: false,

Reviewer: this key can be removed since this will also be supported in k8s

      description:
        'Specifies the request tracing strategy to be used by the underlying http connection manager. Possible values '+
        'are`random_sampling` (default) or `client_sampling`.'
    },
    {
      name: "TracingPercentage",
      type: `float: 0`,
      yaml: false,
      description:

>         'Target percentage of requests received by the gateway that should be traced. When `TracingStrategy` is set ' +
>         'to `client_sampling`, this value is applicable if the `x-client-trace-id` header is set. With ' +
>         '`random_sampling` strategy, this value is applicable when `x-client-trace-id` header is not set.'

Reviewer 2: I think that the description could be a bit clearer. What do you think of this?

```suggestion
        'Target percentage of requests received by the gateway that should be traced. ' + 
        'When `TracingStrategy` is set to `client_sampling` this percentage applies to requests containing the `x-client-trace-id` header. ' +
        'With the `random_sampling` tracing strategy this percentage applies to requests that do not contain the `x-client-trace-id` header.'
```
